src/quantum/azext_quantum/_help.py

```diff
@@ -54,6 +54,9 @@
       - name: Get the list of Azure Quantum workspaces available
         text: |-
             az quantum workspace list
+      - name: Create a new Azure Quantum workspace
+        text: |-
+            az quantum workspace create -g MyResourceGroup -w MyWorkspace -l MyLocation -sa MyStorageAccountName
```

Comment:
Ah, thanks, this is good to know for the other QDK work: the CLI refers to this as "location", not "region". I'll be consistent with that in the other APIs as well.